src/IxMilia.Dxf/Sections/DxfThumbnailImageSection.cs

```diff
@@ -108,7 +108,11 @@ internal static DxfThumbnailImageSection ThumbnailImageSectionFromBuffer(DxfCode
 
                 var section = new DxfThumbnailImageSection();
                 section.Clear();
+#if NET35
+                section.RawData = DxfCommonConverters.HexBytes(string.Join(string.Empty, lines.ToArray()));
```

Comment:
Same comment as `.Cast<T>()`.  I don't see a problem with 'unnecessary' calls to `.ToArray()`.  I'd rather see that extra call than have additional `#if` directives.